run_test.sh

```diff
@@ -374,7 +382,16 @@ EOF
         else
             ALTER_HASH="sha512"
         fi
-        echo "CONFIG_MODULE_SIG_HASH=\"${ALTER_HASH}\"" > /tmp/dkms_test_kconfig
+        if [[ -f "$KCONFIG" ]]; then
+            cp "$KCONFIG" /tmp/dkms_test_kconfig
+            if grep -q '^CONFIG_MODULE_SIG_HASH=' /tmp/dkms_test_kconfig; then
+                sed -i "s/^CONFIG_MODULE_SIG_HASH=.*$/CONFIG_MODULE_SIG_HASH=\"${ALTER_HASH}\"/" /tmp/dkms_test_kconfig
+            else
+                echo "CONFIG_MODULE_SIG_HASH=\"${ALTER_HASH}\"" >> /tmp/dkms_test_kconfig
+            fi
+        else
+            echo "CONFIG_MODULE_SIG_HASH=\"${ALTER_HASH}\"" > /tmp/dkms_test_kconfig
+        fi
```

Comment:
With my MR in mind https://github.com/dell/dkms/pull/312, I think we can split this (+ KCONFIG above) into it's own patch and simplify:
 - assume KCONFIG is always present - error otherwise
 - change, do not set `CONFIG_MODULE_SIG_HASH`